udf_template/custom_udf.py

```diff
@@ -0,0 +1,9 @@
+from abc import ABCMeta
```

Comment:
Not sure about this file? we already have a UDF architecture. I assume this is just a playground file.

Reply:
https://github.com/georgia-tech-db/Eva/pull/36#discussion_r355106834

Reply:
https://github.com/georgia-tech-db/Eva/pull/36#discussion_r355107013